### DISCUSSION TITLE
Update to TanStack/virtual release: v3.0.0-beta.46

### DISCUSSION THREAD

**Discussion by dianventer** [KevinVandy/material-react-table#359] [Ideas]
Would it be possible to update the TanStack/Virtual version to version v3.0.0-beta.46?

There was a massive performance fix in this version in this pull request:
https://github.com/TanStack/virtual/pull/360

We are facing a lot of performance challenges in our application due to this.

**Comment by KevinVandy** [comment]
Eventually when the later virtual betas are stable again, yes.

Also, I'm pretty sure that that PR was before beta 30 so it is already included?

**Reply by dianventer** [reply]
It appears to have been added in 3.0.0-beta.39

https://github.com/TanStack/react-charts/pull/299
